src/dmd/root/longdouble.d

```diff
@@ -83,7 +83,7 @@ struct longdouble_soft
 {
 nothrow @nogc pure:
     // DMD's x87 `real` on Windows is packed (alignof = 2 -> sizeof = 10).
-    align(2) ulong mantissa = 0xC000000000000001UL; // default to snan
+    align(2) ulong mantissa = 0xC000000000000001UL; // default to qnan
```

Comment:
dmd doesn't set the lowest bit, so I think CTFE should not do either, i.e. change the mantissa to 0xC000_0000_0000_0000UL.